AmazonChimeSDK/AmazonChimeSDK/internal/utils/Converters.swift

```diff
@@ -55,10 +55,10 @@ import Foundation
                 return .reconnecting
             case AUDIO_CLIENT_STATE_DISCONNECTING:
                 return .disconnecting
-            case AUDIO_CLIENT_STATE_DISCONNECTED_NORMAL,
-                 AUDIO_CLIENT_STATE_SERVER_HUNGUP:
+            case AUDIO_CLIENT_STATE_DISCONNECTED_NORMAL:
                 return .finishDisconnecting
             case AUDIO_CLIENT_STATE_DISCONNECTED_ABNORMAL,
+                 AUDIO_CLIENT_STATE_SERVER_HUNGUP,
```

Comment:
Same q on android

Reply:
There was confusion about state vs status of audio server hungup.